content/integrations/agents/LINUX-agent/LINUX-standard-install.md

```diff
@@ -8,9 +8,31 @@ author: Lawrence Lane
 #pre: "<i class='fa fa-download'></i> &nbsp;"
 weight: 1
 ---
-This integration’s package (computed metrics, dashboards, and policies that will give you important events and alerts) will be automatically enabled and provisioned to your account as soon as CloudWisdom receives data from the integration. The PACKAGES button on the integration setup page will become active once data is received, so you’ll be able to disable and re-enable the package at will.
+This integration’s package (computed metrics, dashboards, and policies) is automatically enabled and provisioned to your account as soon as CloudWisdom receives data.
 
-## 1. Copy Install Command From Linux Integration Setup Page
+## Cost Optimization Quick Setup
+
+Customers using **only Cost Optimization features**, such as Bill Analysis, can install the Linux Agent with just one quick command! This method enables the Linux Agent's [Simple Collector][2].
```

Comment:
The only report under Bill Analysis relies solely on Cost Explorer and does not use/show utilization metrics from the agent. I would swap "the Right Sizing reports" in for Bill Analysis.